x-pack/plugins/apm/public/plugin.ts

```diff
@@ -138,6 +139,32 @@ export class ApmPlugin implements Plugin<ApmPluginSetup, ApmPluginStart> {
       appRoute: '/app/apm',
       icon: 'plugins/apm/public/icon.svg',
       category: DEFAULT_APP_CATEGORIES.observability,
+      meta: {
+        // !! Need to be kept in sync with the routes in x-pack/plugins/apm/public/components/app/Main/route_config/index.tsx
+        searchDeepLinks: [
+          {
+            id: 'services',
+            title: i18n.translate('xpack.apm.breadcrumb.servicesTitle', {
+              defaultMessage: 'Services',
+            }),
+            path: '/services',
+          },
+          {
+            id: 'traces',
+            title: i18n.translate('xpack.apm.breadcrumb.tracesTitle', {
+              defaultMessage: 'Traces',
+            }),
+            path: '/traces',
+          },
+          {
+            id: 'service-map',
+            title: i18n.translate('xpack.apm.breadcrumb.serviceMapTitle', {
+              defaultMessage: 'Service Map',
+            }),
+            path: '/service-map',
+          },
```

Comment:
If we implement a `getDeepLink` that pulls the necessary info from the routes obejct, we can probably replace this with something like:

```suggestion
		  getDeepLink('services'),
		  getDeepLink('traces'),
		  getDeepLink('service-map'),		  		  

```

Not needed for this PR though.